### PR TITLE
Adding files to java-hamcrest example project

### PR DESCRIPTION
**Pull request by ankit-dq**

### CODE REVIEW

File: Java/hamcrest/src/test/java/deque/exampleTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        driver = new ChromeDriver();
+        axedriver = new AxeDriver(driver);
+        axeSelenium = new AxeSelenium();
+        AxeConfiguration.configure()
+                .testSuiteName("Test")
+                .outputDirectory("axe-json-reports/");
+        Logger =  new File("axe-json-reports/").getAbsolutePath();
+        Destination =  new File("a11y-results/").getAbsolutePath();
+
+    }
```
Comment (Zidious) [inline Java/hamcrest/src/test/java/deque/exampleTest.java:43]:
```suggestion
    private String reporter = new File("src/test/resources/reporter-cli-macos").getAbsolutePath();
    private AxeReportingOptions _reportOptions;
    private WebDriver driver;
    private AxeDriver axedriver;
    private AxeSelenium axeSelenium;
    private String Logger;
    private String Destination;

    @Before
    public void init() {
        _reportOptions = new AxeReportingOptions();
        WebDriverManager.chromedriver().setup();
        driver = new ChromeDriver();
        axedriver = new AxeDriver(driver);
        axeSelenium = new AxeSelenium();
        AxeConfiguration.configure()
                .testSuiteName("Test")
                .outputDirectory("axe-json-reports/");
        Logger =  new File("axe-json-reports/").getAbsolutePath();
        Destination =  new File("a11y-results/").getAbsolutePath();

    }
```

We don't need to use static variables here, we can use Junit's `@Before` which will run before each test. 

We could use JUnit 5 - which has `@BeforeAll` which will run only once (before all tests) and remove the use of `static`

File: Java/hamcrest/src/test/java/deque/exampleTest.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        Runtime rt = Runtime.getRuntime();
+        String command_xml = reporter +" "+ Logger+" --destination "+Destination+" --format xml";
+        String command_html = reporter +" "+ Logger+" --destination "+Destination+" --format html";
+        String command_csv = reporter +" "+ Logger+" --destination "+Destination+" --format csv";
+        try {
+            rt.exec(command_html);
+            rt.exec(command_csv);
+            rt.exec(command_xml);
+        }
+        catch(Exception e){System.out.println(e);}
```
Comment (Zidious) [inline Java/hamcrest/src/test/java/deque/exampleTest.java:69]: nit - 
```suggestion
        catch(Exception e) {
        	e.printStackTrace();
        }
```

File: Java/hamcrest/pom.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        </plugin>
+        <!-- site lifecycle, see https://maven.apache.org/ref/current/maven-core/lifecycles.html#site_Lifecycle -->
+        <plugin>
+          <artifactId>maven-site-plugin</artifactId>
+          <version>3.7.1</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-project-info-reports-plugin</artifactId>
+          <version>3.0.0</version>
+        </plugin>
```
Comment (Zidious) [inline Java/hamcrest/pom.xml:104]: Are we using most of these plugins?

**Review by Zidious — CHANGES_REQUESTED** [review]
Minor changes

File: Java/hamcrest/pom.xml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  <repositories>
+    <repository>
+      <id>deque</id>
+      <url>https://agora.dequecloud.com/artifactory/devtools-maven</url>
+    </repository>
+  </repositories>
+
+  <name>hamcrest</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
```
Comment (Zidious) [inline Java/hamcrest/pom.xml:20]: URL missing here

**Review by Zidious — DISMISSED** [review]
One small thing, LGTM otherwise. Will need @jthickens to review README

**Review by Zidious — APPROVED** [review]
LGTM - @jthickens will need to review the README

File: Java/hamcrest/README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+![logo](./docs/logo-java-hamcrest.png)
+
+# axe DevTools Java Hamcrest API Example
+
+Using axe DevTools Java Hamcrest, you can integrate axe DevTools into your existing testing environment. This example project demonstrates how axe DevTools Java Hamcrest is used to detect accessibility issues and generates reports in CSV, HTML, JSON, and XML formats.
+
+## Prerequisites
+
+- Java 8 or higher
+- Selenium WebDriver
```
Comment (jthickens) [inline Java/hamcrest/README.md:10]: what does selenium webdriver mean here, specifically? A chromedriver binary? Something else?

File: Java/hamcrest/pom.xml [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>31.0.1-jre</version>
+    </dependency>
+    <dependency>
+      <groupId>com.deque.html.axe-devtools</groupId>
```
Comment (jthickens) [inline Java/hamcrest/pom.xml:40]: where's the section to add auth information?

**Review by jthickens — DISMISSED** [review]

File: Java/hamcrest/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+Install the dependencies including **axe DevTools Java Selenium** and **Hamcrest** for the project.
+
+> **_NOTE:_**
+>You need a valid license to use our APIs. For more information, see [Install from Deque’s Agora](https://docs.deque.com/devtools-html/4.0.0/en/java-install-agora) page. After configuring the access to Deque's private registry, you can install the dependencies for this project.
+
+The following command installs all the required dependencies to run this example project.
+
+```sh
+mvn compile
```
Comment (michael-siek) [inline Java/hamcrest/README.md:36]:
```suggestion
mvn clean install
```

**Review by michael-siek — DISMISSED** [review]

**Pull request merged**